src/mem/Allocator.h

```diff
@@ -72,6 +139,8 @@ class Allocator : public Interface
 
 private:
     const char *label = nullptr;
+    const size_t objectSize_ = 0;
```

Comment:
We may have discussed this before, but I find constants with two values odd and constants with bogus values confusing/dangerous. I suggest making both data members constant and rely on the constructor for their proper initialization:

```suggestion
    const char * const label;
    const size_t objectSize_;
```